### PR TITLE
feat: support negative syntax with ### on img2img

### PR DESCRIPTION
**Pull request by jaszczw**
Same implementation of negative prompts support as on txt2img

### CODE REVIEW

**Comment by db0** [comment]
Maybe I'm stupid, but where is it actually using the negprompt?

File: nataili/inference/compvis/img2img.py [hunk 1]
```diff
@@ -631,7 +639,8 @@ def sample(init_data, x, conditioning, unconditional_conditioning, sampler_name)
                         prompts = all_prompts[n * batch_size : (n + 1) * batch_size]
                         seeds = all_seeds[n * batch_size : (n + 1) * batch_size]
 
-                        uc = model.get_learned_conditioning(len(prompts) * [""])
+
```
Comment (jaszczw) [inline nataili/inference/compvis/img2img.py:642]: @db0 isn't that the line, I basically hacked my way by looking at sd-webui code

File: nataili/inference/compvis/img2img.py [hunk 2]
```diff
@@ -631,7 +639,8 @@ def sample(init_data, x, conditioning, unconditional_conditioning, sampler_name)
...
+                        uc = model.get_learned_conditioning(len(prompts) * [negprompt])
```
Comment (jaszczw) [inline nataili/inference/compvis/img2img.py:643]: @db0 isn't that the line, I basically hacked my way by looking at sd-webui code

Comment (jaszczw) [reply nataili/inference/compvis/img2img.py:643]: I'll, review some other repositories, but we have same code on txt2img.

Comment (db0) [reply nataili/inference/compvis/img2img.py:643]: OK I saw the len and I thought it's doing just a claculation, I don't understand how it work but fine. Lemme check with @hlky

**Pull request merged**